toolboxes/scripts/GRGUtilities.py

```diff
@@ -569,8 +569,8 @@ def GRGFromArea(AOI,
 
 
 def GRGFromPoint(starting_point,
+				 vertical_cells,
```

Comment:
I'm not sure this is the best way to fix this since this second parameter remains "Number of Horizontal Grid Cells" elsewhere in the code, ex:

https://github.com/Esri/military-tools-geoprocessing-toolbox/blob/dev/toolboxes/scripts/GRGTools.py#L366
https://github.com/Esri/military-tools-geoprocessing-toolbox/blob/dev/toolboxes/scripts/GRGTools.py#L221

So it might confuse the maintainers long term when this parameter changes from "horizontal" to "vertical" from the calling method to this one.  It might be a better longer term fix to fix where these variable are reversed in the code below in this method:

https://github.com/Esri/military-tools-geoprocessing-toolbox/blob/dev/toolboxes/scripts/GRGUtilities.py#L696

Reply:
I see your point; however, whenever I have tried to change those inputs I get unequal grids in height and width when I expect them to be equal. I have tried all the different combinations and I cannot get the same expectation that I get from passing the parameters in switched as they were with the first pull request. 
@csmoore what would you recommend?

Reply:
@topowright - the variables appear to be switched in this section of the code (ex. vertical cells incorrectly used to set width vs. horizontal cells): https://github.com/Esri/military-tools-geoprocessing-toolbox/blob/dev/toolboxes/scripts/GRGUtilities.py#L696 - I can add this fix to the PR if you like